src/main/scala/ie/zalando/fabric/gateway/models/SynchDomain.scala

```diff
@@ -228,6 +229,16 @@ object SynchDomain {
     val skipperStringValue: String = s"""compress($factor, "$encoding")"""
   }
 
+  object DynamicFiler {
```

Comment:
Typo, should be DynamicFilter

Reply:
Arrrggghhhh. I was so careful checking all my words in teh docs... :(